src/libstd/sys/windows/path.rs

```diff
@@ -19,6 +19,10 @@ pub fn is_verbatim_sep(b: u8) -> bool {
     b == b'\\'
 }
 
+pub fn is_valid_drive_letter(disk: u8) -> bool {
```

Comment:
The name speaks for itself, but if you could add a doc comment with the link to some Windows standard explaining this, that would be fantastic <3